sdk/lib/rtl/path.c

```diff
@@ -2160,6 +2166,44 @@ NTSTATUS NTAPI RtlNtPathNameToDosPathName(IN ULONG Flags,
     return STATUS_SUCCESS;
 }
 
+/*
+ * @implemented
+ */
+NTSTATUS
+NTAPI
+RtlSetSearchPathMode(IN ULONG Flags)
```

Comment:
```suggestion
RtlSetSearchPathMode(_In_ ULONG Flags)
```

Reply:
Is this change necessary? It only makes the difference in coding style. Does it change anything else?

Reply:
What is the point of using old-style `IN` in *.c, while using new-style `_In_` in *.h?

Reply:
As I see, such style is used for all other functions in these *.c and *.h, so I just use the general style.

Reply:
Obviously not in the 2 modified .h...

Reply:
OK, committed.